gocd/vagrant-publisher.opensuse.gocd.yaml

```diff
@@ -0,0 +1,22 @@
+format_version: 3
+pipelines:
+  Vagrant.Publish:
+    group: Vagrant
```

Comment:
If you introduce a new group, you won't see the pipeline. The access rights are per group - so please don't

Reply:
Is the new group name ok? I am not sure how gocd handles sub-groups.

Reply:
I don't think there is something like a sub-group. the group we have is called openSUSE.checkers - and openSUSE users can operate it - and all Users can see it. By default new groups are only visible to admins - I don't think that's what you want.

Reply:
Hm, so what would you suggest that I use as the group?

Reply:
Pretty much everything uses `openSUSE.Checkers`

Reply:
`openSUSE.Checkers` it is then.